raiden-ts/src/helpers.ts

```diff
@@ -556,37 +557,37 @@ export async function getState(
 
 /**
  * For a given channelId (passed as opts.meta), waits for a deposit to be confirmed and for the
- * following messageGlobalSend for its PFSCapacityUpdate to succeed
+ * following messageServiceSend for its PFSCapacityUpdate to succeed
```

Comment:
This is just a representative line for the whole second commit (96670eb48255218f7d5dce622128612c956ad80d). 
Why was this replacement necessary? I can only guess and argue. But it would be cool if I as a reviewer must not make assumptions but can just follow your decisions. Especially because this commit is not just about renaming but there are some minor logic tweaks. I don't know the exact reason for both... :wink:

Reply:
Beforehand, the services messages were sent to the respective **global** rooms (and the epics which emitted those actions knew about it). Now, those rooms don't exist anymore, and the epics must instead emit actions specifying to which service they want to send the messages to, and the transport is the one who now knows how to get those messages to the services (now, through toDevice messages to the specific set of userIds from services from `ServicesRegistr`y). The renaming comes with this logic change, because the perspective also changed slightly. From the changes only, it may be a little harder to see that, but if you're reading the actions and code for the first time, hopefully this is what you'll get from the actions signature and usage.

Reply:
Perfect. Next time a version of this in the commit message. Please. :pray: :upside_down_face:

Reply:
:heavy_check_mark: